test/e2e/kubectl.go

```diff
@@ -32,22 +32,18 @@ import (
 )
 
 const (
-	nautilusImage       = "kubernetes/update-demo:nautilus"
-	kittenImage         = "kubernetes/update-demo:kitten"
-	updateDemoSelector  = "name=update-demo"
-	updateDemoContainer = "update-demo"
-	kubectlProxyPort    = 8011
+	nautilusImage            = "kubernetes/update-demo:nautilus"
+	kittenImage              = "kubernetes/update-demo:kitten"
+	updateDemoSelector       = "name=update-demo"
+	updateDemoContainer      = "update-demo"
+	frontendSelector         = "name=frontend"
+	redisMasterSelector      = "name=redis-master"
+	redisSlaveSelector       = "name=redis-slave"
+	kubectlProxyPort         = 8011
+	guestbookResponseTimeout = 2 * time.Minute
```

Comment:
I would bump this to 10 minutes. This timeout includes waiting for multiple pods/services/replication controllers to be created. The default pod creation timeout we use is 5 minutes.